### PR TITLE
[WIP] Feature/CV2-4629: Managing articles in item page

### PR DESCRIPTION
**Pull request by danielevalverde**
## Description

- Implement the components for displaying explainers and fact-checks
- When an item has articles, the default state for the left sidebar should be a list of these articles
- implement new fragment and fix mutations when adding an article
- Make the ItemReportStatus, LastRequestDate, and Language components more flexible to be reusable on this feature
- Add the edit button
- Add the fact-check disclaimer using the existing <Alert /> component

Missing TODO:
-  Implement the different behaviors for when there is only fact-check, only explainers or both
- Fix CSS to match design
- integrate the edit button with CV2-4630
- add unit tests

References: CV2-4629

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?
Go to the item page and check that fact checks and explainers are displayed correctly.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Comment by brianfleming** [comment]
@danielevalverde I think you need to repoint this PR to the epic branch

**Comment by caiosba** [comment]
Done, @brianfleming

File: src/app/components/media/MediaComponent.js [hunk 1]
```diff
@@ -266,7 +266,6 @@ export default createFragmentContainer(withPusher(MediaComponent), graphql`
     project_id
     last_seen
     demand
-    articles_count
```
Comment (caiosba) [inline src/app/components/media/MediaComponent.js:269]: Question: Are you sure you can delete this? It was being used for the Articles tab.

**Review by caiosba — CHANGES_REQUESTED** [review]
I left some review comments and questions, there are no blockers here IMO (except for a couple of strings that need to be localized)... I may fix those myself to get it merged to the epic branch soon.

File: src/app/components/media/MediaComponent.js (hunk 1 above)
Comment (caiosba) [reply src/app/components/media/MediaComponent.js:269]: Yes, it's needed. I just put it back.

File: src/app/components/article/MediaArticlesDisplay.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        <MediaArticleCard
+          key={factCheck.id}
+          variant="fact-check"
+          title={factCheck.title || factCheck.claim_description?.description}
+          url={factCheck.url}
+          languageCode={factCheck.language !== 'und' ? factCheck?.language : null}
+          date={factCheck.updated_at}
+          statusColor={currentStatus ? currentStatus.style?.color : null}
+          statusLabel={currentStatus ? currentStatus.label : null}
+          publishedAt={factCheck?.claim_description?.project_media?.report_status === 'published' && factCheck?.claim_description?.project_media?.published ? parseInt(factCheck?.claim_description?.project_media?.published, 10) : null}
```
[the remark this answers is not in the document]
Comment (caiosba) [reply src/app/components/article/MediaArticlesDisplay.js:31]: Fixed.

File: src/app/components/article/MediaArticles.js [hunk 3]
```diff
@@ -155,10 +142,10 @@ MediaArticlesComponent.propTypes = {
...
     dbid: PropTypes.number.isRequired,
     type: PropTypes.string.isRequired,
-    fact_check: PropTypes.object,
-    explainers: PropTypes.exact({
-      edges: PropTypes.arrayOf(PropTypes.object),
-    }),
+    // fact_check: PropTypes.object,
+    // explainers: PropTypes.exact({
+    //   edges: PropTypes.arrayOf(PropTypes.object),
+    // }),
```
[the remark this answers is not in the document]
Comment (caiosba) [reply src/app/components/article/MediaArticles.js:148]: Removed.

**Comment by caiosba** [comment]
I applied the things from code review and also fixed some other things I found on the way. I implemented the styles to match the design and also implemented the different behaviors for when it has fact-check, explainers or both. Functionality for the "edit article" button will be implemented in a different ticket.

**Pull request merged**